src/consts/models.tsx

```diff
@@ -426,8 +431,17 @@ export type RootStackParams = {
   };
   [Route.CreateTransactionPassword]: undefined;
   [Route.ConfirmTransactionPassword]: { setPassword: string };
-  [Route.LocalConfirmNotificationCode]: { children: React.ReactNode; onSuccess: () => void; title: string };
+  [Route.LocalConfirmNotificationCode]: {
+    children: React.ReactNode;
+    onSuccess: () => void;
+    title: string;
+    onResend: () => void;
```

Comment:
already added resend in my PR, anyway as it is written below we don't need local confirmation for deleting the email without wallets so you don't have to modify this screen at all in this PR